src/scenes/ProgressReports/EditForm/Steps/Media/VariantMediaAccordion.tsx

```diff
@@ -0,0 +1,105 @@
+import { AddPhotoAlternate, ExpandMore } from '@mui/icons-material';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  IconButton,
+} from '@mui/material';
+import { useState } from 'react';
+import { RoleIcon } from '~/components/RoleIcon';
+import { MediaInfoForm } from './MediaInfoForm';
+import { MediaVariantResponse } from './MediaStep';
+
+const ImageBox = ({
+  media,
+  openFileBrowser,
+}: {
+  media: any[];
+  openFileBrowser: any;
+}) => {
+  return (
+    <>
+      {media.length ? (
+        // TODO replace with image when it exists
+        <Box
+          sx={{
+            borderRadius: '3px',
+            backgroundColor: 'rgba(0, 0, 0, 0.12)',
+            p: 1,
+            pt: 6,
+            m: 1,
+            flexGrow: 1.75,
+            textAlign: 'center',
+          }}
+        >
+          <img src={media[0].media.file.url} alt={media[0].media.altText} />
+        </Box>
+      ) : (
+        <Box
+          sx={{
+            borderRadius: '3px',
+            backgroundColor: 'rgba(0, 0, 0, 0.12)',
+            p: 1,
+            pt: 6,
+            m: 1,
+            flexGrow: 1.75,
+            textAlign: 'center',
+          }}
+        >
+          <IconButton onClick={openFileBrowser}>
+            <AddPhotoAlternate sx={{ height: 48, width: 48 }} />
+          </IconButton>
+        </Box>
+      )}
+    </>
+  );
+};
+
+// TODO proper types
+export const VariantMediaAccordion = ({
+  openFileBrowser,
+  response,
+  initiallyOpen,
+  updateMedia,
+}: {
+  openFileBrowser: any;
+  response: MediaVariantResponse;
+  initiallyOpen: boolean;
+  updateMedia: any;
+}) => {
+  const [expanded, setExpanded] = useState(initiallyOpen);
+
+  return (
+    <Accordion key={response.uploadVariant.key} expanded={expanded} square>
+      <AccordionSummary
+        aria-controls={`${response.uploadVariant.key}-content`}
+        expandIcon={<ExpandMore />}
+        sx={{
+          '& .MuiAccordionSummary-content': {
+            alignItems: 'center',
+          },
+        }}
+        onClick={() => setExpanded(!expanded)}
+      >
+        <RoleIcon variantRole={response.uploadVariant.responsibleRole} />
+        <span>{response.uploadVariant.label}</span>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Box sx={{ display: 'flex', p: '0px 16px' }}>
```

Comment:
```suggestion
      <AccordionDetails sx={{ display: 'flex', px: 2 }}>
```

Reply:
This right here is proof I need a new laptop sooner rather than later. WS showed an issue with this, so I added the box. Today, I recently closed and re-opened WB and this works....